packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.ts

```diff
@@ -31,10 +57,31 @@ export function trackViewMetrics(
     viewStart,
     (newLoadingTime) => {
       viewMetrics.loadingTime = newLoadingTime
+
+      // We compute scroll metrics at loading time to ensure we have scroll data when loading the view initially
+      // This is to ensure that we have the depth data even if the user didn't scroll or if the view is not scrollable.
+      if (isExperimentalFeatureEnabled(ExperimentalFeature.SCROLLMAP)) {
+        const { scrollHeight, scrollDepth, scrollTop } = computeScrollMetrics()
+
+        scrollMetrics = {
+          scrollHeight,
+          maxDepth: scrollDepth,
+          maxDepthTime: newLoadingTime,
+          scrollTop,
+        }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏scrollMetrics naming is representing two different things which is a bit confusing.
should it be `scrollMetrics` and `maxDepthMetrics`? 
or with the future introduction of some kind of fold metrics, should it be `scrollValues` and `scrollMetrics`?